lib/Rest/HttpResponse.h

```diff
@@ -84,6 +84,10 @@ class HttpResponse : public GeneralResponse {
                   bool resolve_externals = true) override;
   void addRawPayload(velocypack::StringRef payload) override;
 
+  virtual bool isResponseEmpty() const override {
```

Comment:
```suggestion
 bool isResponseEmpty() const override {
```
only parent class needs to be virtual